src/controllers/user.controller.js

```diff
@@ -132,4 +132,26 @@ export default class UserController {
       Response.UnauthorizedError(res, { message: 'Unable to sign in' });
     }
   }
+
+  /**
+   * Handles auto retrieval of user's profile needed for travel
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} response
+   */
+  static retrieveProfile(req, res) {
+    const { id: user_id } = req.body.user;
+    const { rememberMe } = req.query;
```

Comment:
This `remember me` appears to be for caching. If so, it is sleek! I've never thought of it this way.